vit_keras/layers.py

```diff
@@ -143,10 +177,16 @@ def call(self, inputs, training):
         y = self.layernorm2(x)
         y = self.mlpblock(y)
         return x + y, weights
-
+        
```

Comment:
@faustomorales I can remove it but it may look a bit congested, moreover, previously whitespace was used before the `get_config()` method so I simply followed it. You can check how may it look if I remove the whitespace.

![image](https://user-images.githubusercontent.com/36858976/123519040-c0b8b380-d6ca-11eb-883b-34d43c92d269.png)